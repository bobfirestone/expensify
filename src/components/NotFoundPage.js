import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <div>
    404! No Bueno <Link to="/">Go Home. You're Drunk!</Link>
  </div>
);

export default NotFoundPage;
