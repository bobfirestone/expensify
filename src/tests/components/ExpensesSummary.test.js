import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test("should should render 0 items & $0 when no expenses are given", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={0} expensesTotal={0} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should render given a single expense", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={1} expensesTotal={100} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should render given multiple expenses", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={3} expensesTotal={356} />
  );
  expect(wrapper).toMatchSnapshot();
});
