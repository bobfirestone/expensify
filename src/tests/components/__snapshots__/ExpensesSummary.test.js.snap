// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render given a single expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $1.00
  </h1>
</div>
`;

exports[`should render given multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totaling 
    $3.56
  </h1>
</div>
`;

exports[`should should render 0 items & $0 when no expenses are given 1`] = `
<div>
  <h1>
    Viewing 
    0
     
    expenses
     totaling 
    $0.00
  </h1>
</div>
`;
